wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java

```diff
@@ -123,6 +124,34 @@ public void close() {
     LiveWindow.setDisabledListener(null);
   }
 
+  /**
+   * Starts publishing added epochs to NetworkTables.
```

Comment:
Very this, `startDataLog(DataLog)`, and `startDataLog(DataLog, String)`, this is the only description of what is actually published. Maybe adding something to the names (e.g., `publishEpochToNetworkTables` or `startWatchdogDataLog`) would help?

Reply:
I'm going to use the `DriverStation` route and just keep calling it `startDataLog`, but instead add documentation on what's logged. I think it would be good to drill into a team's head the idea that `startDataLog` or `publishToNetworkTables` is the definitive way to get access to debug info. Right now, `startDataLog` is something not widespread within the library; I think establishing some kind of consistency is a good idea. If this isn't a good API shape, we can always go back and change things.

Reply:
My main concern was that this could be misleading since it implies that everything is logged, when in fact only the epochs are. I agree that establishing consistent names to get access to debug info is important, but I'd like for those names to consistently mean "start reporting all useful data about the state of this thing". For example, if I randomly saw something like `CommandScheduler.getInstance().startDataLog(dataLog, "CommandScheduler");`, I personally would think it starts logging currently running commands and whether the scheduler's disabled, not that it logs loop timings.